cmd/wallet-web/src/components/Vaults/VaultCard.vue

```diff
@@ -23,7 +23,7 @@
           <img class="w-6 h-5" src="@/assets/img/vaults.svg" alt="Vault Icon" />
         </div>
         <div class="px-3 md:px-0 pb-4">
-          <span class="block pt-4 text-lg font-bold text-neutrals-dark"> {{ name }}</span>
+          <span :id="id" class="block pt-4 text-lg font-bold text-neutrals-dark"> {{ name }}</span>
```

Comment:
You should set the `id` to the `app-link` element instead, since it handles the click event